CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 ### Features
 - [#814] Implement `org-roam-insert-immediate`
+- [#839] Return selected file from `org-roam-insert`
```

Comment:
Should I use a link to this pull request?

Reply:
Yup, that would be good!